docs/envvars.rst

```diff
@@ -74,6 +74,13 @@ applicable.
       - xonsh.environ.FORMATTER_DICT  
       - Dictionary containing variables to be used when formatting PROMPT and TITLE 
         see `Customizing the Prompt <tutorial.html#customizing-the-prompt>`_.
+    * - HISTCONTROL
+      - ``''``
+      - A comma-separated list of options that determine if commands are saved
```

Comment:
There are a bunch of examples in environ.py and tools.py, but let me know if you have any questions.